engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -176,6 +176,13 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
             // the log.
             if (vrProvider.init()) {
                 playerCamera = new OpenVRStereoCamera(vrProvider);
+                /*
+                * The origin of OpenVR's coordinate system lies on the ground of the user. We have to move this origin
+                * such that the ground plane of the rendering system and the ground plane of the room the VR user is
+                * in match.
+                 */
+                //
+                vrProvider.getState().setGroundPlaneYOffset(-0.80f - context.get(Config.class).getPlayer().getEyeHeight());
```

Comment:
I've now put in what is essentially a fudge factor here. The issue is that if I move `getEyeHeight()` units down from the camera height, I don't encounter the ground plane. To me this doesn't seem right.